### ISSUE TITLE
Rework Material You implementation inspired by Vitamin implementation

### ISSUE THREAD

**Issue by GerardPaligot** [GerardPaligot/conferences4hall#65]
After the integration of the new Design System Vitamin, Material You implementation should be updated to match features and UI of Vitamin implementation which is so much better than M2 and M3 implementations.
